### PR TITLE
New digest

### PR DESCRIPTION
**Pull request by levity**
less coupling, ~200 fewer LOC, more tests 🎉

### CODE REVIEW

File: lib/community/digest2/sampleData.json [hunk 1]
```diff
@@ -0,0 +1,97 @@
+{
```
Comment (levity) [inline lib/community/digest2/sampleData.json]: this isn't used in the code; it's a copy of the sample data in SendWithUs

**Comment by levity** [comment]
@robbiecarlton please take a look

**Comment by levity** [comment]
i sent you a test email with sample data

**Comment by robbiecarlton** [comment]
couple of things I noticed in the test email (you're probably already aware):

No community name with the logo
No icons in the reply or view buttons
Reply and view buttons don't do anything when clicked

screen shot 2016-06-29 at 11 24 42 am
screen shot 2016-06-29 at 11 25 01 am

**Comment by levity** [comment]
yep, no worries. #1 and #2 are pending design & assets from Damian, #3 is
working but i didn't put the links in the sample data.

**Comment by robbiecarlton** [comment]
👍  looks good!

**Pull request merged**